packages/dev/core/src/Culling/ray.core.ts

```diff
@@ -24,6 +24,25 @@ export type MeshPredicate = (mesh: AbstractMesh, thinInstanceIndex: number) => b
  */
 export type TrianglePickingPredicate = (p0: Vector3, p1: Vector3, p2: Vector3, ray: Ray, i0: number, i1: number, i2: number) => boolean;
 
+/**
+ * This class allows user to customize internal picking mechanism
+ */
+export class PickingCustomization {

```

Comment:
Can be an object (const), no need for the overhead of a class with a single static member

Reply:
yes but I want to be ready for the future as we may add more customizations

Reply:
You can add more to an interface, and just create an object implementing it

Reply:
but somehow it is not unique
What is the point then?

Reply:
Put differently, why bothering about 2 things (interface + implementation) when a class works?

Reply:
Answering my own question: There is a slight overload with classes because of prototype